src/pages/_app.tsx

```diff
@@ -31,7 +31,7 @@ interface ExplorerAppProps extends AppProps {
 
 function ExplorerApp({ Component, ...rest }: ExplorerAppProps): React.ReactElement {
   const { apiServer, networkMode, pageProps } = rest;
-  const { isHome, fullWidth } = pageProps;
+  const { isHome, fullWidth, ...restPageProps } = pageProps;
```

Comment:
The other page props needed to be forwarded to the component for `getStaticProps` to work